site2/docs/security-kerberos.md

```diff
@@ -366,19 +362,19 @@ And add a section of `Client` configurations in the file `pulsar_jaas.conf`, whi
 };
 ```
 
-In this setting, Pulsar Broker's principal and keyTab file indicates Broker's role when authenticating with ZooKeeper.
+In this setting, the principal of Pulsar Broker and keyTab file indicates the role of Broker when you are authenticating with ZooKeeper.
 
 ## Regarding authentication between BookKeeper and Broker
 
-Pulsar Broker acts as a Kerberos client when authenticating with Bookie. According to [BookKeeper document](http://bookkeeper.apache.org/docs/latest/security/sasl/), you need to add `bookkeeperClientAuthenticationPlugin` parameter in `broker.conf`:
+Pulsar Broker acts as a Kerberos client when you are authenticating with Bookie. According to [BookKeeper document](http://bookkeeper.apache.org/docs/latest/security/sasl/), you need to add `bookkeeperClientAuthenticationPlugin` parameter in `broker.conf`:
```

Comment:
Replace "when you are v-ing" with "when you v xxx".
Use present tense in technical writing. Check all similar cases.